diff_drive_controller/include/diff_drive_controller/diff_drive_controller.h

```diff
@@ -336,20 +342,6 @@ namespace diff_drive_controller
     /// and differentiation scheme, which can be: "analytic", "autodiff"
     std::string integrate_method_;
     std::string integrate_differentiation_;
-  private:
```

Comment:
This `private` is superfluous; there's another one on L169: https://github.com/clearpathrobotics/ros_controllers/blob/7d04ca23aaf444e1eb2214c10b6ff9177422d139/diff_drive_controller/include/diff_drive_controller/diff_drive_controller.h#L169